docs/user/user/metrics/adding-new-metrics.md

```diff
@@ -20,45 +20,45 @@ The following is a set of questions to ask about the data being collected to hel
 
 If you need to know how many times something happened relative to the number of times something else happened,
-use a [rate metric](metrics/rate.html).
+use a [rate metric](../../reference/metrics/rate.html).
 
-If you need to know when the things being counted happened relative to other things, consider using an [event](metrics/event.html).
+If you need to know when the things being counted happened relative to other things, consider using an [event](../../reference/metrics/event.html).
 
 ### Are you measuring time?
 
-If you need to record an absolute time, use a [datetime metric](metrics/datetime.html). Datetimes are recorded in the user's local time, according to their device's real time clock, along with a timezone offset from UTC. Datetime metrics allow specifying the resolution they are collected at, and to stay [lean][lean-data], they should only be collected at the minimum resolution required to answer your question.
+If you need to record an absolute time, use a [datetime metric](../../reference/metrics/datetime.html). Datetimes are recorded in the user's local time, according to their device's real time clock, along with a timezone offset from UTC. Datetime metrics allow specifying the resolution they are collected at, and to stay [lean][lean-data], they should only be collected at the minimum resolution required to answer your question.
 
 If you need to record how long something takes you have a few options.
 
-If you need to measure the total time spent doing a particular task, look to the [timespan metric](metrics/timespan.html). Timespan metrics allow specifying the resolution they are collected at, and to stay [lean][lean-data], they should only be collected at the minimum resolution required to answer your question.
+If you need to measure the total time spent doing a particular task, look to the [timespan metric](../../reference/metrics/timespan.html). Timespan metrics allow specifying the resolution they are collected at, and to stay [lean][lean-data], they should only be collected at the minimum resolution required to answer your question.
 Note that this metric should only be used to measure time on a single thread. If multiple overlapping timespans are measured for the same metric, an invalid state error is recorded.
 
-If you need to measure the relative occurrences of many timings, use a [timing distribution](metrics/timing_distribution.html). It builds a histogram of timing measurements, and is safe to record multiple concurrent timespans on different threads.
+If you need to measure the relative occurrences of many timings, use a [timing distribution]../..(/reference/metrics/timing_distribution.html). It builds a histogram of timing measurements, and is safe to record multiple concurrent timespans on different threads.
```

Comment:
```suggestion
If you need to measure the relative occurrences of many timings, use a [timing distribution](../../reference/metrics/timing_distribution.html). It builds a histogram of timing measurements, and is safe to record multiple concurrent timespans on different threads.
```

Reply:
It's interesting that links checker doesn't break with this. Either mdbook is inteligent enough to compile correctly even when this is wrong or links checker is doing something different.